agcaballero/portfolio/src/main/webapp/script.js

```diff
@@ -72,6 +72,129 @@ function testRandomPost(trials = 10000) {
 
 testRandomPost(10000);  // runs 10000 trials on randomPost()
 
+/** Creates a Map using Map API */
+function createMap() {
```

Comment:
Nit: consider being more specific in the name, since this script file has a broad scope, for example, `createPhotoLocationMap`.

Reply:
That makes sense. I agree.

Reply:
Done